lib/util/git/url.ts

```diff
@@ -15,6 +15,7 @@ export function getHttpUrl(url: string, token?: string): string {
     ? parsedUrl.protocol
     : 'https';
 
+  parsedUrl.user = parsedUrl.password = null;
   parsedUrl.token = token ?? '';
```

Comment:
While `parsedUrl` extracts the user + password it doesn't allow setting them. Just setting `user` to `''` should be enough.

```suggestion
  parsedUrl.user = '';
```

Reply:
Based on your comment, the diff should be applied on L18, right?

```diff
-parsedUrl.user = parsedUrl.password = null;
+parsedUrl.user = '';
 parsedUrl.token = token ?? '';
```

Reply:
sorry, yeah. Added a suggestion at the right place.